packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -127,15 +126,15 @@ function useMovingAnimation(
 		reset: triggeredAnimation !== finishedAnimation,
 		config: { mass: 5, tension: 2000, friction: 200 },
 		immediate: prefersReducedMotion,
-		onFrame: ( props ) => {
+		onFrame: () => {
 			if (
 				adjustScrolling &&
 				scrollContainer.current &&
-				! prefersReducedMotion &&
-				props.y
```

Comment:
Is this check no useful anymore? I can't remember why I introduced it. I guess maybe the idea is to not scroll if the vertical position don't change.